cluster/examples/kubernetes/1.5/rook-operator.yaml

```diff
@@ -23,3 +29,13 @@ spec:
         # current mon with a new mon (useful for compensating flapping network).
         - name: ROOK_MON_OUT_TIMEOUT
           value: "300s"
+        - name: ROOK_AGENT_IMAGE_TAG
+          value: master
+        - name: ROOK_SYSTEM_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: NODE_NAME
```

Comment:
It doesnt. This is for the agents to know which nodes they are running on. That is needed for the mounting

Reply:
You are right but the operator also needs to know its node so that it can query its API for kubelet config. We use this information to auto discover where the flexplugin are supposed to go.

Reply:
what kubelet config is needed? seems odd to look at the local node to answer a question for other nodes

Reply:
yeah.. this would not work on non-uniform nodes. So all kubelets on all nodes have to be configured the same way. Which is the case in google and probably the case in 99% of the deployments.